ibis/backends/risingwave/__init__.py

```diff
@@ -106,48 +105,6 @@ def _from_url(self, url: ParseResult, **kwargs):
 
         return self.connect(**kwargs)
 
-    def _register_in_memory_table(self, op: ops.InMemoryTable) -> None:
```

Comment:
This was a duplicated definition not caught by ruff, and that I introduced in #10669.